### PR TITLE
Update Desu Source. Add genres and improve paging

### PR DESCRIPTION
**Pull request by Pavkazzz**
Update russian source desu.me

### CODE REVIEW

File: src/ru/desu/build.gradle [hunk 1]
```diff
@@ -5,9 +5,9 @@ ext {
     appName = 'Tachiyomi: Desu'
     pkgNameSuffix = "ru.desu"
     extClass = '.Desu'
-    extVersionCode = 2
+    extVersionCode = 3
     extVersionSuffix = 1
-    libVersion = '1.0'
+    libVersion = '1.1'
```
Comment (Taumer) [inline src/ru/desu/build.gradle]: 1.2

File: src/ru/desu/build.gradle [hunk 2]
```diff
@@ -5,9 +5,9 @@ ext {
     appName = 'Tachiyomi: Desu'
     pkgNameSuffix = "ru.desu"
     extClass = '.Desu'
-    extVersionCode = 2
+    extVersionCode = 3
     extVersionSuffix = 1
```
Comment (Taumer) [inline src/ru/desu/build.gradle]: Update this too

**Comment by Pavkazzz** [comment]
Done

**Comment by Pavkazzz** [comment]
Merge please

**Pull request merged**